test/query_test.exs

```diff
@@ -610,16 +610,16 @@ defmodule QueryTest do
 
   @tag min_pg_version: "9.0"
   test "encode network types", context do
-    assert [[%Postgrex.INET{address: {127, 0, 0, 1}, netmask: 32}]] =
-           query("SELECT $1::inet", [%Postgrex.INET{address: {127, 0, 0, 1}, netmask: 32}])
-    assert [[%Postgrex.INET{address: {8193, 43981, 0, 0, 0, 0, 0, 0}, netmask: 128}]] =
-           query("SELECT $1::inet", [%Postgrex.INET{address: {8193, 43981, 0, 0, 0, 0, 0, 0}, netmask: 128}])
+    assert [[%Postgrex.INET{address: {127, 0, 0, 1}, netmask: nil}]] =
```

Comment:
The tests in this function should output text instead of the structs - the reverse of the decode test. This ensure we only test encode or decode at one time to avoid hiding bugs on both sides. If want to do identity tests have that in a separate test. It is an existing issue with this test.
It should be like:
```elixir
assert query("SELECT $1::inet::text", [%Postgrex.INET{address: {127, 0, 0, 1}, netmask: nil}]) == [["127.0.0.1/32"]]
```

Reply:
I can rewrite these tests, but the only difference for these particular tests is to clear the netmask.